### PR TITLE
Add captions and headers to data tables

### LINKED ISSUE #2936: VPAT -- Layout Tables (Customize Website Theme)
`<table>` is typically used to present information in tabular form, but is also commonly used for page layout.  For accessibility, the two can be distinguished by the use, or lack of, table headers.

Data tables should have `<th>`, presumably also enclosed with `<thead>` ... `</thead>`.  Also a `<caption>` should be provided before `<thead>`.  Or possibly use `<caption />` if there is no caption to display.

Layout tables do not have explicit headings or caption in HTML.  Therefore, they are identified by the absence of `<thead>`, `<th>`, and `<caption>`.

The Customize Website Theme view of Course Configuration appears to have a _layout_ table (for uploading HTML files) but table headers (`<th>`) is being used for visual styling for boldface fonts.  As this table seems to have more to do with layout than data, boldface visual styling should be enforced with CSS, not with HTML table headers.

Per WAVE regarding **Data** Tables
> **What It Means**
> A data table is present.
> **Why It Matters**
> Data tables present tabular data. Data tables should contain table header cells that identify the content of their respective row and/or columns. Tables with proper table headers provide additional information and navigation for screen reader users.
> **How to Fix It**
> Ensure that the table contains tabular data and that it is not used merely for page layout. Ensure that all column and row headers are `<th>` elements and ensure the data cells are associated with their proper header cells (typically by assigning scope to the table headers). Where appropriate, associate a descriptive caption (`<caption>` element) to the table.
> **The Algorithm... in English**
> A `<table>` element is present that contains at least one table header cell (`<th>`).
> **Standards and Guidelines**
> [Section 508 (g)](http://webaim.org/standards/508/checklist#standardg)
> [Section 508 (h)](http://webaim.org/standards/508/checklist#standardh)
> [1.3.1 Info and Relationships (Level A)](http://webaim.org/standards/wcag/checklist#sc1.3.1)

Per WAVE regarding **Layout** Tables
> **What It Means**
> A layout table is present.
> **Why It Matters**
> While tables are primarily intended for the presentation of tabular information or data, they are often used to control page layout and formatting. Layout tables can introduce reading and navigation order issues and must not contain header (`<th>`) cells.
> **How to Fix It**
> Ensure that the table is indeed a layout table and that it does not contain tabular data. If it is a data table, provide appropriate header (`<th>`) cells. Verify that the reading and navigation order of table content (based on underlying source code order) is logical.
> **The Algorithm... in English**
> A `<table>` element is present that does not contain any header (`<th>`) cells.
> **Standards and Guidelines**
> None

### PR DESCRIPTION
**Pull request by J16053**
Closes #2936 

Continuing off of @drumilpatel2000 's work in #3336 , I have refactored the rest of the tables to conform with the specification as per WAVE.

### CODE REVIEW

**Comment by J16053** [comment]
@pbailie could you please review and let me know if you have any suggestions?

File: site/app/templates/admin/users/RotatingSectionsForm.twig [hunk 1]
```diff
@@ -132,8 +149,15 @@
                         </table>
                     </div>
                     <div class="col-md-6">
-                        <h3>Users with<br>Registration Section=NULL</h3>
+                        {# This is a data table #}
                         <table class="table table-bordered table-striped">
+                            <caption>Users with Registration Section = NULL</caption>
```
[the remark this answers is not in the document]
Comment (pbailie) [reply site/app/templates/admin/users/RotatingSectionsForm.twig]: Resolved in commit 6107ae5

File: site/app/templates/admin/users/RotatingSectionsForm.twig [hunk 2]
```diff
@@ -121,8 +131,15 @@
                 <h2>Current Rotating Section Counts</h2>
                 <div class="row">
                     <div class="col-md-6">
-                        <h3>Registered Students<br>(non NULL registration section)</h3>
+                        {# This is a data table #}
                         <table class="table table-bordered table-striped">
+                            <caption>Registered Students (non-NULL registration section)</caption>
```
[the remark this answers is not in the document]
Comment (pbailie) [reply site/app/templates/admin/users/RotatingSectionsForm.twig]: Resolved in commit 6107ae5

File: site/app/templates/admin/admin_gradeable/rubric/CheckpointsRubric.twig [hunk 3]
```diff
@@ -1,11 +1,13 @@
 <div class="multi-field-wrapper-checkpoints">
+    {# This is a data table #}
```
[the remark this answers is not in the document]
Comment (pbailie) [reply site/app/templates/admin/admin_gradeable/rubric/CheckpointsRubric.twig]: Resolved in commit 6107ae5

File: site/app/templates/admin/UploadWrapperForm.twig [hunk 4]
```diff
@@ -22,14 +22,18 @@
             <div class="wrapper-bottom">Bottom Bar</div>
         </div>
     </div>
-	<br>
+    <br>
+    {# This is a data table #}
```
[the remark this answers is not in the document]
Comment (pbailie) [reply site/app/templates/admin/UploadWrapperForm.twig]: Resolved in commit 6107ae5

File: site/app/templates/admin/admin_gradeable/rubric/NumericRubric.twig [hunk 5]
```diff
@@ -6,53 +6,57 @@
     <br /> <br />
 
     <div class="multi-field-wrapper-numeric">
-        <h5>Numeric Items</h5>
+        {# This is a data table #}
```
[the remark this answers is not in the document]
Comment (pbailie) [reply site/app/templates/admin/admin_gradeable/rubric/NumericRubric.twig]: Per internal meeting with @bmcutler, this change is being reverted in commit a3079ec

File: site/app/templates/admin/admin_gradeable/rubric/NumericRubric.twig [hunk 6]
```diff
@@ -6,53 +6,57 @@
...
         <table class="numerics-table table table-bordered" style=" border: 1px solid #AAA; max-width:50% !important;">
             <!-- Headings -->
+            <caption style="text-align: left">Numeric Items</caption>
```
[the remark this answers is not in the document]
Comment (pbailie) [reply site/app/templates/admin/admin_gradeable/rubric/NumericRubric.twig]: Per internal meeting with @bmcutler, this change is being reverted in commit a3079ec

File: site/app/templates/admin/admin_gradeable/rubric/NumericRubric.twig [hunk 7]
```diff
@@ -6,53 +6,57 @@
...
+                    </td>
+                </tr>
+            </tbody>
         </table>
-
-        <h5>Text Items</h5>
+        <br />
+        {# This is a data table #}
         <table class="text-table table table-bordered" style=" border: 1px solid #AAA; max-width:25% !important;">
+            <caption style="text-align: left">Text Items</caption>
```
[the remark this answers is not in the document]
Comment (pbailie) [reply site/app/templates/admin/admin_gradeable/rubric/NumericRubric.twig]: Per internal meeting with @bmcutler, this change is being reverted in commit a3079ec

File: site/app/templates/admin/admin_gradeable/rubric/NumericRubric.twig [hunk 8]
```diff
@@ -6,53 +6,57 @@
...
+                    <td>
+                        <input type="checkbox" name="numeric_extra_0" class="numeric_extra extra" value="" onchange="calculateTotalScore();"/>
+                    </td>
+                </tr>
+            </tbody>
         </table>
-
-        <h5>Text Items</h5>
+        <br />
+        {# This is a data table #}
```
[the remark this answers is not in the document]
Comment (pbailie) [reply site/app/templates/admin/admin_gradeable/rubric/NumericRubric.twig]: Per internal meeting with @bmcutler, this change is being reverted in commit a3079ec

**Pull request merged**